Integrations/CentralBankIntegrationLib/CentralBankNewsPublisher.cs

```diff
@@ -20,8 +19,19 @@ public string GetNews()
             List<Valute> valutes = GetTopValutes();
 
             var sb = new StringBuilder();
-            sb.AppendLine("Top - 5 exchange rates:");
-            sb.AppendLine(valutes != null ? string.Join('\n', valutes) : Errors.CanNotGetCurrencyRateFromCB);
+            if (valutes.Count == 0)
+            {
+                sb.AppendLine("Valutes are not founded");

```

Comment:
Why Valute, not Currency?
"founded" must be replaced with "found".
Or even the whole string: "No currencies found".

Reply:
Ok